engine/src/execution_graph/logic_controllers/LogicalProject.cpp

```diff
@@ -101,6 +102,22 @@ std::unique_ptr<cudf::column> evaluate_string_functions(const cudf::table_view &
         computed_col = cudf::strings::contains_re(column, regex);
         break;
     }
+    case operator_type::BLZ_STR_REPLACE:
+    {
+        // required args: string column, search, replacement
+        assert(arg_tokens.size() == 3);
+        RAL_EXPECTS(!is_literal(arg_tokens[0]), "REPLACE function not supported for string literals");
+
+        cudf::column_view column = table.column(get_index(arg_tokens[0]));
+        RAL_EXPECTS(is_type_string(column.type().id()), "REPLACE argument must be a column of type string");
+
+        // remove the raw single quotes
+        std::string target = arg_tokens[1].substr(1, arg_tokens[1].size() - 2);
```

Comment:
You should validate that the string is indeed encapsulated by single quotes. I am assuming it usually will, but in case int he future we change something up, we would want to make sure we are not stripping the first and last characters of the string of interest. 
You can also use the string utility `removeEncapsulation` from StringUtils.h

Reply:
Thanks for the tip. Should other parts of the codebase be using this utility as well? The pattern `arg_tokens[1].substr(1, arg_tokens[1]` comes up quite a bit in LogicalProject

Reply:
Yeah, it would be best to use the utility more in the codebase

Reply:
Sounds good. I'll probably update the rest of this file then

Reply:
Resolved by https://github.com/BlazingDB/blazingsql/pull/1193/commits/da60b5de0420bb6b6a08cdd7d5aa1bfbcbb70c0e